### DISCUSSION TITLE
Upgrade to 2.3.160 Kibana Error

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#8735] [General]
Hello, 
I upgraded to 2.3.160 and now when i launch Kibana it gives a 404 error. I orginally also had 500 errors loading alerts and dashboards too. I have tried the following: reboot, so-status ( all are ok) , restart kibana, restart soc. None of these worked. Anyone have any recommendations to try?

**Comment by bnlalippman** [comment]
I received a 404 error on Kibana when upgrading from 2.3.130 to 2.3.140. This was because we needed elastic auth enabled, and it was not. 

Check if this command gives you "False" in the output from the manager.
`
salt-call pillar.get elasticsearch:auth
`

If it does, you may need to enable elastic authentication.
`
so-elastic-auth true
`

If you have Security Onion support I would definitely recommend reaching out first, but this is what worked for me.

**Reply by cerberus10748** [reply]
I had a false reply, I tried the other command you said and still have the same 404 error.

**Comment by dougburks** [comment]
When you upgraded, did you follow all steps at https://docs.securityonion.net/en/2.3/soup.html#elastic-8?

**Reply by cerberus10748** [reply]
I looked over this and tried restarting several services and to no avail i still get 404 error. Is there a way to revert security onion or do i need to do a clean install?

**Reply by dougburks** [reply]
You can do a clean install.

**Comment by cerberus10748** [comment]
Doug, 
Sadly I was bad and didnt follow that when I upgraded and just did a sudo soup update. I will check out those steps and guide to see if there is anything I can get out of it. Thank you .
